src/components/LoginPage/LoginPageComponent.js

```diff
@@ -30,11 +30,37 @@ const H3 = styled.h3`
   color: ${COLORS.white};
 `;
 
+const GoogleLoginButton = styled.button`
+  color: ${COLORS.white};
+  background-color: ${COLORS.blue};
+  float: right;
+  position: relative;
+  border: none;
+  border-radius: 0.4vmin;
+  padding: 2vmin 4vmin 2vmin calc(9vmin + 2%);
+
+  img {
+    height: 92%;
+    background-color: ${COLORS.white};
+    position: absolute;
+    padding: 3%;
+    left: 0.75%;
+    top: 4%;
+    border-radius: 0.4vmin;
```

Comment:
The styles were copy-pasted from lines 72-79

<img width="687" alt="copypaste" src="https://user-images.githubusercontent.com/6686679/82252650-106d6d00-9958-11ea-9240-ef35356b001a.png">

Could you please make those styles reusable instead? :)
I.e. to have them coded only once and used for both buttons.